step-3-adding-some-tests.md

```diff
@@ -41,7 +45,7 @@ CREATE TABLE IF NOT EXISTS talks(
 
 INSERT
   INTO talks (id, title)
-  VALUES ('welcome-to-junit-5', 'JUnit 5.2 actually :)')
+  VALUES ('welcome-to-junit-5', 'JUnit 5.x actually :)')
```

Comment:
FYI the reason why it was `5.2` is because that was a reference to a talk from Spring I/O, and, with `5.x`, and makes very little sense - perhaps we should use something less weird then? :)

Reply:
shall we just put a Testcontainers talk in there? :) 
`Modern Integration Testing with Testcontainers`

Reply:
Yep, let's change 👍